src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -291,27 +454,36 @@ TEST_F(HMICapabilitiesTest, LoadCapabilitiesFromFile) {
       display_capabilities_so[hmi_response::text_fields].length();
   EXPECT_NE(0u, text_len);
   for (uint32_t i = 0; i < text_len; ++i) {
-    EXPECT_TRUE((display_capabilities_so[hmi_response::text_fields][i])
+    const int32_t index = static_cast<int32_t>(i);
+    EXPECT_TRUE((display_capabilities_so[hmi_response::text_fields][index])
                     .keyExists(strings::name));
-    EXPECT_TRUE((display_capabilities_so[hmi_response::text_fields][i])
+    EXPECT_TRUE((display_capabilities_so[hmi_response::text_fields][index])
                     .keyExists(strings::character_set));
   }
 
   // Check image fields
-  EXPECT_TRUE((display_capabilities_so).keyExists(hmi_response::image_fields));
-  const uint32_t img_len =
+  ASSERT_TRUE((display_capabilities_so).keyExists(hmi_response::image_fields));
+  const uint64_t img_len =
```

Comment:
If you assume that `img_len` can be `uint64_t` does this mean that over loop can be bellow because of `for (uint32_t i = 0; i < img_len; ++i)`?